elementary/monitor/data_monitoring/data_monitoring_alerts.py

```diff
@@ -30,17 +32,64 @@ def __init__(
         self,
         config: Config,
         tracking: AnonymousTracking,
+        filter: Optional[str] = None,
         force_update_dbt_package: bool = False,
         disable_samples: bool = False,
         send_test_message_on_success: bool = False,
     ):
-        super().__init__(config, tracking, force_update_dbt_package, disable_samples)
+        super().__init__(
+            config, tracking, force_update_dbt_package, disable_samples, filter
+        )
+        self.filter = self._parse_filter(self.raw_filter)
         self.alerts_api = AlertsAPI(
-            self.internal_dbt_runner, self.config, self.elementary_database_and_schema
+            self.internal_dbt_runner,
+            self.config,
+            self.elementary_database_and_schema,
         )
         self.sent_alert_count = 0
         self.send_test_message_on_success = send_test_message_on_success
 
+    def _parse_filter(
+        self, filter: Optional[str] = None
+    ) -> Optional[DataMonitoringAlertsFilter]:
+        if filter:
+            if self.user_dbt_runner:
```

Comment:
You still need to handle the owner selector in this case as well, no?
As it's not a supported dbt one

Reply:
@elongl showed me a very cool trick that works for dbt selector:
`dbt ls -s config.meta.owner:idan`

We decided that for now we will only document the non dbt project path, and if a user will ask for advance selector we will refer him to the dbt selector and will explain how to use it 🙂

Reply:
So that's cool, but I still don't like that the interfaces for dbt project and non-dbt project are not the same. It means that things will break if you move to the dbt project mode.

So in my mind either you:
1. Support `owner:` in both.
2. Change the interface of the second to be: `config.meta.owner:`